example/src/example_poor_kv_handler.erl

```diff
@@ -1,67 +1,49 @@
 -module(example_poor_kv_handler).
```

Comment:
This file was not only formatted but adapted to work with the `cowboy` version `trails` expects.